doc/sphinx-guides/source/installation/r-rapache-tworavens.rst

```diff
@@ -516,7 +510,7 @@ edited:**
 
 2. ``hostname="localhost:8080";``
 
-   changed to point to the dataverse app, from which TwoRavens will be obtaining the metadata and data files. (don't forget to change 8080 to the correct port number!)
+   changed to point to the Dataverse installation app, from which TwoRavens will be obtaining the metadata and data files. (don't forget to change 8080 to the correct port number!)
```

Comment:
Not sure is "app" is necessary here with the addition of "installation".

Reply:
fixed in https://github.com/IQSS/dataverse/pull/7567/commits/f41355f317062479dce6cb4d02f716459681e8cc